app/views/bookings/index.html.erb

```diff
@@ -1,31 +1,44 @@
-<p id="notice"><%= notice %></p>
+<div class="container">
+  <ol class="breadcrumb">
+    <li><%= link_to "Home Work", root_path %></li>
+    <li class="active">Bookings</li>
+  </ol>
+  <% if notice.present?  %>
+    <div class="alert alert-success" role="alert" id="notice"><%= notice %>
+      <button type="button" class="close" data-dismiss="alert" aria-label="Close">
+        <span aria-hidden="true">&times;</span>
+      </button>
+    </div>
+  <% end %>
 
-<h1>Listing Bookings</h1>
+  <div class="table-responsive">
+    <table class="table table-hover table-bordered">
+      <thead>
+        <tr>
+          <th>#</th>
+          <th>Customer</th>
+          <th>Cleaner</th>
+          <th>Date</th>
+          <th colspan="3">Actions</th>
+        </tr>
+      </thead>
 
-<table>
-  <thead>
-    <tr>
-      <th>Customer</th>
-      <th>Cleaner</th>
-      <th>Date</th>
-      <th colspan="3"></th>
-    </tr>
-  </thead>
+      <tbody>
+        <% @bookings.each_with_index do |booking, index| %>
+        <tr>
+          <td><%= index + 1 %></td>
+          <td><%= booking.customer.first_name+" "+booking.customer.last_name %></td>
```

Comment:
@keyurgohil2426 Should use string interpolation `#{..}` instead of usual concatenation.